### DISCUSSION TITLE
今天我的遠端桌面設定的跳病毒了 想知道為甚麼

### DISCUSSION THREAD

**Discussion by 123hi123** [moesnow/March7thAssistant#296] [Q&A]
使用的是作者推薦的遠端桌面設置方法
倒不是說真的覺得有毒
只是好其裡面塞了甚麼被判斷有病毒
有毒的文件是
照片中的superRDP.dll
![image](https://github.com/user-attachments/assets/20410d9c-eaba-490b-b52d-4edbd3881cc7)
跳出的病毒顯示如下
https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3AWin32%2FMalgent&threatid=2147794830
通知我找不到了
然後附上下載的地方
![image](https://github.com/user-attachments/assets/b6304c48-a1c3-42bb-89fe-df8688da8b5a)
![image](https://github.com/user-attachments/assets/27a67ea3-8fdb-40e7-9681-8a8f2262d024)

歡迎大家來討論

**Comment by GripenANM** [comment]
https://www.virustotal.com/gui/file/d5f2f0f05dd40cbb6d71c0fa19ed55e5903f2e2001124b6d43494d2df1201cfa/detection
